src/tests/Interop/COM/ComWrappers/GlobalInstance/GlobalInstance.TrackerSupport.cs

```diff
@@ -31,15 +31,16 @@ static int Main(string[] doNotUse)
                 // The first test registers a global ComWrappers instance for tracker support.
                 // Subsequents tests assume the global instance has already been registered.
                 ValidateRegisterForTrackerSupport();
-
+#if Windows
```

Comment:
The managed tests are built once for all platforms in CI. Can you convert these `#if Windows` blocks to `if (OperatingSystem.IsWindows())` blocks instead?